app/views/categories/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="top-nav py-3">
+  <div class= "container d-flex align-items-center">
+    <%= link_to categories_path, class: 'd-flex text-decoration-none text-white' do %>
+      <i class="fa fa-arrow-left fs-3 " aria-hidden="true"></i>
+    <% end %>
+    <h1 class= "container d-flex justify-content-center text-white">DETAILS</h1>
+  </div>
+</div>
+<div class="container mt-3 py-3  bg-light  ">
+<div class = "bg-white shadow-lg p-3 mb-5 rounded">
+  <div id="img-div" class="col d-flex gap-2 border-bottom border-light border-4">
+    <img src=<%= @category.icon%> alt=<%= @category.name%> />
+    <p class = "fw-bold text-secondary fs-5 d-flex align-items-center"><%= @category.name %></p>
+  </div>
+  <div>
+    <div id="payments" >
+      <% @payments.each do |payment| %>
+      <%= link_to "/categories/#{@category.id}/payments/#{payment.id}", class: "text-decoration-none" do %>
+      <div class="container border-bottom border-light border-4 mt-3">
+        <div><p class = "fw-light text-info fs-5">Transaction <%= payment.name %></p></div>
+        <div class="d-flex align-items-baseline gap-2 ">
+        <i class="fa fa-calendar" aria-hidden="true"></i>
+        <p><%= payment.created_at.to_date.strftime('%Y %b %d') %> at <%= payment.created_at.to_time.strftime('%H:%M:%S') %></p>
+        </div>
+       </div>
+      <% end %>
+  <% end %>
+</div>
+  </div>
+  <div class="d-flex justify-content-between">
+    <p class = "fw-bold text-secondary fs-5">TOTAL PAYMENT</p>
+    <p class = "fw-bold text-dark fs-5">$<%= @category.count_total_payments %></p>
+  </div>
+  <div class="my-5 d-flex justify-content-center">
+    <%= link_to "New transaction", new_payment_path, class: "btn btn-primary" %>
+  </div>
+  </div>
+</div>
```

Comment:
- [ ] As mentioned by the previous reviewer to stick as much as possible to the [template design from Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), I think this page design should be improved to match the expected design. This should look something like the image below(e.g.: fix the width && height of the category icon, correctly align all elements to their expected position, use the  correct fonts,...):
![Screenshot from 2022-12-02 15-59-37](https://user-images.githubusercontent.com/83042742/205310721-8c86a7fe-b9f0-476f-9211-d2922150586a.png)